### PR TITLE
Unify apps/simple and examples - move everything to examples

### PR DESCRIPTION
**Pull request by marcusschiesser**
After this PR, all `simple` examples are in the `examples/` folder and can be used with the local build and the published package. 
I made a diff between `apps/simple` and `examples` to unify the examples.

### CODE REVIEW

File: examples/pg-vector-store/load-docs.ts
Comment (marcusschiesser) [inline examples/pg-vector-store/load-docs.ts:1]: @yisding the pg-vector-store example is not working as we don't the store in `llamaindex`. Can be fixed in a separate PR

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -135,15 +135,15 @@ importers:
         specifier: ^11.1.0
         version: 11.1.0
       llamaindex:
-        specifier: workspace:*
-        version: link:../../packages/core
+        specifier: latest
+        version: link:../packages/core
```
Comment (marcusschiesser) [inline pnpm-lock.yaml:139]: @yisding this is pretty cool: If we would keep this line, the `examples` package would always use the local version (when using `pnpm`).
As we might overwrite this by accident, I added the `pnpm link` command to the readme anyhow

File: examples/pg-vector-store/load-docs.ts
Comment (yisding) [reply examples/pg-vector-store/load-docs.ts:1]: Thanks for the catch!

File: examples/README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
 ```
+
+Then run the examples with `ts-node`, for example `npx ts-node vectorIndex.ts`
+
+## Using the local build
+
+```bash
+pnpm install
+pnpm --filter llamaindex build
+pnpm link ../packages/core
```
Comment (yisding) [inline examples/README.md:26]: Nice!

**Review by yisding — APPROVED** [review]
Awesome, thanks for doing this Marcus!

**Pull request merged**